app/frontend/components/domains/permit-application/status-filter.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button, Checkbox, CheckboxGroup, Collapse, CaretDown, Divider, Flex, Funnel } from "@chakra-ui/react"
+import React, { useEffect, useMemo, useState } from "react"
+import { useTranslation } from "react-i18next"
+import { ISearch } from "../../../lib/create-search-model"
+import { ESortDirection } from "../../../../frontend/types/enums"
+import { IOption, ISort } from "../../../../frontend/types/types"
+
+interface ISearchSortProps {
+  searchModel: ISearch
+  i18nPrefix: string
+  sortFields: string[]
+  isOpen: boolean
+}
+
+export const StatusFilter: React.FC<ISearchSortProps> = ({ searchModel, i18nPrefix, sortFields, isOpen }: ISearchSortProps) => {
```

Comment:
Since we have decided that the permit application status is something that will be separate from these filter, perhaps it is best to call this PermitApplicationFiltersMenu
be sure to also change the filename

Reply:
@bkmaibach This file has been replaced with the permit-application-filters-menu.tsx